ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -62,9 +62,16 @@ TENANT_ID=$(jq -r .tenantId "${SHARED_DIR}/osServicePrincipal.json")
 cat >> "${SHARED_DIR}/azurestack-login-script.sh" << EOF
 
 if [[ -f "${CLUSTER_PROFILE_DIR}/ca.pem" ]]; then
-  cp "${CLUSTER_PROFILE_DIR}/ca.pem" /tmp/ca.pem
-  cat /usr/lib64/az/lib/python*/site-packages/certifi/cacert.pem >> /tmp/ca.pem
-  export REQUESTS_CA_BUNDLE=/tmp/ca.pem
+    cp "${CLUSTER_PROFILE_DIR}/ca.pem" /tmp/ca.pem
+    if ls /usr/lib64/az/lib/python*/site-packages/certifi/cacert.pem > /dev/null 2>&1; then
```

Comment:
I wonder why not use `-f` here?

Reply:
Because the file path contains wildcards "*", in this way, we don't care about python version.
Checked that ` if [[ -f /tmp/ca* ]]; then echo "ok"; fi` didn't work yesterday. But tried again just now on local host, ` if [ -f /tmp/ca* ]; then echo "ok"; fi` works, let me update to use this condition.

Reply:
I guess you are right about the file path contains wildcards "*":
```
[jianlinliu@jialiu-laptop installer]$ ls ./D*
./Dockerfile.ci  ./Dockerfile.ci.rhel7  ./Dockerfile.upi.ci  ./Dockerfile.upi.ci.rhel8
[jianlinliu@jialiu-laptop installer]$ if [ -f Dockerfile.upi.* ]; then echo "ok"; fi
bash: [: Dockerfile.upi.ci: binary operator expected
```

Reply:
hmm, `[ ]` works on my local host, checked step `upi-install-azurestack` in [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/54195/rehearse-54195-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-azure-stack-upi-f28/1811264411780780032), also works well, no error this time.
```
~/installer /output
WARNING: Switched active cloud to 'WWT'.
WARNING: Use 'az login' to log in to this cloud.
WARNING: Use 'az account set' to set the active subscription.
```

Reply:
if more than one files are matched, indeed get some issues. It should be more safe to use `ls <file_path> > /dev/null 2>&1`
```
[jima@mylaptop tmp]$ ls *.sh
azurestack-login-script-1.sh  azurestack-login-script.sh  cucushift-installer-check-azure-cp-identity-commands.sh  test.sh
[jima@mylaptop tmp]$ if [ -f *.sh ]; then echo "ok"; fi
bash: [: too many arguments
[jima@mylaptop tmp]$ if ls *.sh > /dev/null 2>&1; then echo "ok"; fi
ok

```

Reply:
Okay, if the wildcards path only locate one file, then `[]` works.

Reply:
okay, I am okay to restore to use `ls`

Reply:
change back to use `ls <file_path> > /dev/null 2>&1`.
PTAL